item.go

```diff
@@ -9,6 +9,10 @@ const (
 	// NoTTL indicates that an item should never expire.
 	NoTTL time.Duration = -1
 
+	// PreviousTTL indicates that existing TTL of item should be used
+	// default TTL will be used as fallback if item doesn't exist
+	PreviousTTL time.Duration = -2
```

Comment:
I think `PreviousOrDefaultTTL` would be more descriptively accurate. In case we would decide to expand this further, we could also add `PreviousOrNoTTL` (but no need for now).

Reply:
> I think `PreviousOrDefaultTTL` would be more descriptively accurate. In case we would decide to expand this further, we could also add `PreviousOrNoTTL` (but no need for now).

Yes that makes sense, I have pushed new changes.